.spelling

```diff
@@ -29,6 +29,18 @@ DaemonSet
 DaemonSets
 StatefulSet
 StatefulSets
+VictoriaMetrics
+Vmagent
+Vmalert
+Vminsert
+Vmselect
+Vmstorage
+vmstorage
+vmalert
+vmselect
+victoria
+kubectl
+kubelet
```

Comment:
Several of these look like command line calls. Command calls shouldn't be added to the `.spelling` file, instead wrap each offense with backticks (`).